react-app/src/containers/StartScreen/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react';
+
+import styles from './index.css';
+
+import Logo from 'components/Logo';
+import Button from 'components/Button';
+import InputText from 'components/InputText';
+
+export default class StartScreen extends PureComponent {
+  render() {
+    return (
+      <div className={styles.container}>
+        <Logo size={Logo.BIG} />
+        <InputText
+          type={InputText.TEXT}
+          label="Username: "
+          placeholder="Username"
+          onChange={event => console.log(`Username: ${event}`)}
```

Comment:
Instead of using a placeholder function you should go ahead and create a proper function to handle the value changes on the input components.
For now, use some placeholder validators (e.g. non-empty) to verify the inputs have data on them and enable/disable the `Button`s
Save the inputs contents on this component's state, we will need that data when we make this component do something when the buttons are pressed.
Also, add a handler to focus the password input when enter is pressed on the username input.

Reply:
Done.